application-diagram-ui/src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -1171,6 +1171,46 @@ define('diagram-editor', [
     }
   };
 
+
+  /*
+   * Map with all the menu items that we want to have a title.
+   */
+
+  const titleMap = new Map([
+      ['print', 'When you click this button, a popup will appear, allowing you to select which part of the diagram you want to print or export as a PDF.\
+        After clicking the "Print button, a new window will open, displaying a preview of the document. In this window, you can choose to either print the document or save it as a PDF']
+  ]);
+
+/**
+ * Modify the original method a bit to allow us to add titles to menu items.
+ */
+  Menus.prototype.addMenuItem = function(menu, key, parent, trigger, sprite, label)
+  {
+      var action = this.editorUi.actions.get(key);
+      var addTitle = titleMap.has(key);
+      if (action != null &amp;&amp; (menu.showDisabled || action.isEnabled()) &amp;&amp; action.visible)
+      {
+          var item = menu.addItem(label || action.label, null, function(evt)
+          {
+              action.funct(trigger, evt);
+          }, parent, sprite, action.isEnabled());
+          if (addTitle)
+          {
+            item.title = titleMap.get(key);
+          }
+          // Adds checkmark image
+          if (action.toggleAction &amp;&amp; action.isSelected())
+          {
+              menu.addCheckmark(item, Editor.checkmarkImage);
+          }
+
+          this.addShortcut(item, action);
+
+          return item;
+      }
+
+      return null;
+  };
```

Comment:
Instead of duplicating code and adding what you need,  here you can use the returned `item`. Is this working?

```suggestion
  var originalAddMenuItem = Menus.prototype.addMenuItem;
  Menus.prototype.addMenuItem = function(menu, key, parent, trigger, sprite, label) {
      let item = originalAddMenuItem.apply(this, arguments);
      if (item != null && titleMap.has(key)) {
        item.title = titleMap.get(key);
      }
      return item;
  };
```

Reply:
Initially, I approached it differently and couldn’t override the method because I had made changes at various points in the function. However, I later realized that my solution was flawed and overlooked the fact that I could simply use the returned item with the current implementation.